src/components/Blocks/ProjectDetails/ProjectDetails.tsx

```diff
@@ -62,18 +54,27 @@ export function ProjectDetails({ links, toolsUsed, contributors }: IProjectDetai
                 </div>
                 <div className={styles.text}>
                     <h2 className="h1">Contributors</h2>
-                    <ul className={styles.contributors}>
-                        {contributors.map(({ name, image, linkUrl }, index) => (
-                            <li key={name + index}>
-                                <SmartLink to={linkUrl}>
-                                    <div className={styles.image}>
-                                        <Image src={image.src} height="85" width="85" alt={image.src} />
-                                    </div>
-                                    <h3 className="p">{name}</h3>
-                                </SmartLink>
-                            </li>
-                        ))}
-                    </ul>
+                    {Array.isArray(contributors) && contributors.length > 0 ? (
+                        <ul className={styles.contributors}>
+                            {contributors.map(({ login, avatar_url, html_url }) => (
+                                <li key={login}>
+                                    <SmartLink to={html_url}>
+                                        <div className={styles.image}>
+                                            <Image
+                                                src={avatar_url}
+                                                height="85"
+                                                width="85"
+                                                alt={`The github profile picture for ${login}`}
+                                            />
+                                        </div>
+                                        <h3 className="p">{login}</h3>
+                                    </SmartLink>
+                                </li>
+                            ))}
+                        </ul>
+                    ) : (
+                        <div>There was an error loading the contributors</div>
```

Comment:
How about we don't show there is an error to a user? I'd think a user probably wouldn't care/notice if the list isn't showing, but may raise an eyebrow if they see an error happened.

Reply:
Good point, I'll just trim the whole contributors section if the api call fails.